src/gui/sdl_mapper.cpp

```diff
@@ -1754,41 +1764,135 @@ static void change_action_text(const char* text,Bit8u col) {
 	bind_but.action->SetColor(col);
 }
 
+static std::string humanize_mod_name(const CBindList &binds, const std::string &fallback)
+{
+	auto trim_pfx = [](const std::string &pfx, const std::string &input) {
+		if (starts_with(pfx, input))
+			return input.c_str() + pfx.size();
```

Comment:
Took  me a while to realize `pfx` was prefix. I originally though it was a function name, with `fx` being a common short-hand. 

I'd prefer we use the full word, `trim_prefix` or `cut_prefix`.. it would let the calls below read more naturally.

Reply:
This function changed a bit, but I used the name `trim_prefix`, as it still applies (when used in this context).